app/models/manageiq/providers/base_manager/refresher.rb

```diff
@@ -16,7 +16,7 @@ def initialize(targets)
 
       def options
         return @options if defined?(@options)
-        @options = VMDB::Config.new("vmdb").config[:ems_refresh]
+        @options = Settings.ems_refresh
```

Comment:
Does this change affect the other managers?  Some might be doing fetch_path which isn't available on a ::Config::Options class.